include/options/abstract-option.php

```diff
@@ -227,9 +199,18 @@ protected function prepare( $value ) {
 	 * @return mixed The sanitized value. An instance of `WP_Error` in case of blocking error.
 	 */
 	protected function sanitize( $value, Options $options ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
-		return rest_sanitize_value_from_schema( $value, $this->get_schema(), $this->key() );
+		return rest_sanitize_value_from_schema( $value, $this->get_specific_schema(), static::key() );
 	}
 
+	/**
+	 * Returns the default value.
+	 *
+	 * @since 3.7
+	 *
+	 * @return mixed
+	 */
+	abstract protected function get_default();
```

Comment:
```suggestion
	abstract protected function default();
```
To follow the same namgin as for `key()`, what do you think?

Reply:
We also have `get_schema()`, `get_errors()`, and `get_specific_schema()`/`get_data_structure()`.
So...?